helm-charts/infisical/templates/backend-deployment.yaml

```diff
@@ -9,23 +10,23 @@ metadata:
   labels:
     {{- include "infisical.backend.labels" . | nindent 4 }}
 spec:
-  replicas: {{ .Values.backend.replicaCount }}
+  replicas: {{ $backend.replicaCount }}
   selector:
     matchLabels:
       {{- include "infisical.backend.matchLabels" . | nindent 6 }}
   template:
     metadata: 
       labels:
         {{- include "infisical.backend.matchLabels" . | nindent 8 }}
-      {{- with .Values.backend.podAnnotations }}
+      {{- with $backend.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
     spec:
       containers:
-      - name: {{ template "infisical.name" . }}-{{ .Values.backend.name }}
-        image: "{{ .Values.backend.image.repository }}:{{ .Values.backend.image.tag | default .Chart.AppVersion }}"
-        imagePullPolicy: {{ .Values.backend.image.pullPolicy }}
+      - name: {{ template "infisical.name" . }}-{{ $backend.name }}
+        image: "{{ $backend.image.repository }}:{{ $backend.image.tag | default .Chart.AppVersion }}"
```

Comment:
I'm wondering if we should get rid of the `default .Chart.AppVersion`.Because we would never have a image tag that is the `.Chart.AppVersion`. Those aren't related. What do you think

Reply:
Yes, perfect. Let's do the same for frontend as well